Dockerfile_mbtci_template

```diff
@@ -72,12 +82,15 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
       gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys "$key" || \
       gpg --batch --keyserver hkps://keyserver.ubuntu.com --recv-keys "$key" ; \
     done \
-    && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz" \
-    && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+    && curl -fsSLO --compressed "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-${GOOS}-${ARCH}.tar.xz" \
+    && curl -fsSLO --compressed "https://nodejs.org/dist/v${NODE_VERSION}/SHASUMS256.txt.asc" \
     && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
-    && grep " node-v$NODE_VERSION-linux-$ARCH.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
-    && tar -xJf "node-v$NODE_VERSION-linux-$ARCH.tar.xz" -C /usr/local --strip-components=1 --no-same-owner \
-    && rm -rf "$GNUPGHOME" "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
+    && grep " node-v${NODE_VERSION}-${GOOS}-${ARCH}.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
+    && mkdir -p ${NODE_HOME} \
+    && tar -xJf "node-v${NODE_VERSION}-${GOOS}-${ARCH}.tar.xz" -C ${NODE_HOME} --strip-components=1 \
+    && chmod --recursive a+w "${NODE_HOME}" \
```

Comment:
Security issue: no user need to access the node.js installation path.

Reply:
chmod --recursive a+w "${NODE_HOME}" is added because of a customer issue https://github.com/SAP/cloud-mta-build-tool/issues/1018

In the case, customer exec "npm link @sap/cds-dk --local" in /home/mta folder, it will raise a access problem , such as :"permission denied, mkdir '/usr/local/lib/node_modules".

In my commit, I install nodejs in "/usr/local/nodejs", and to solve the customer's problem, I set the nodejs folder "a+w"  right

Reply:
1. the issue was already solved by the use of `--no-same-owner ` and the correct fix is not chmod 777 but chown 
2. fixing bug unrelated to the PR purpose must be done in another PR
3. installation path change is not the purpose of the PR and must be done in another PR
4. chmod 777 on directory is not compliant with the SAP security standards 

And I've already explained at large in several comments why 3 is not necessary with in depth rationale.

Reply:
I revert the code which change owner for /usr/local/lib/node_modules path from PR #1023 in my commit https://github.com/SAP/cloud-mta-build-tool/pull/1023/commits/c449c37d3318ae82808dd931157c37c0315b41c5 , and this problem should be fixed in your PR #1026 

@rodibrin